src/components/Input/index.tsx

```diff
@@ -0,0 +1,55 @@
+import type { HTMLAttributes, ReactElement } from 'react'
+import { ChattingInput } from './ChattingInput'
+import { DefaultInput } from './DefaultInput'
+import { EditInput } from './EditInput'
+import { SearchInput } from './SearchInput'
+
+type InputStyle = typeof INPUT_STYLE_KEYS[keyof typeof INPUT_STYLE_KEYS]
+export type InputSize = 'large' | 'small'
+export interface InputProps extends HTMLAttributes<HTMLInputElement> {
+  inputStyle?: InputStyle
+  inputSize?: InputSize
+  label?: string
+  status?: 'none' | 'success' | 'error' | 'default'
+  message?: string
```

Comment:
```suggestion
  message?: string
```
message 보다는 `noticeMessage` 또는 `guideMessage` 와 같이 어떤 역할을 하는 메시지인지 명시되면 좋을 것 같습니다!

Reply:
[요기](dc4e0b948281b0d69dfbda51ee92ad4c4dbaad47)에 반영 완료 ! ☺️
guideMessage로 주영넴과 통일했읍니다